improver/cli/phase_probability.py

```diff
@@ -41,14 +41,14 @@ def process(*cubes: cli.inputcube, radius: float = 10000.0):
     probability of a specific precipitation phase being found at the surface.
 
     Args:
-        cubes (iris.cube.CubeList or list):
+        cubes:
```

Comment:
Whilst it is not important, this is the only CLI in which the type declarations have been removed, so maybe put them back so the CLIs remain self-consistent.

Reply:
This must have been by mistake. I've reverted any changes to `improver/cli/*`.